servicetalk-http-netty/src/test/java/io/servicetalk/http/netty/H2ResponseCancelTest.java

```diff
@@ -168,8 +168,8 @@ private void requestCancellationResetsStreamButNotParentConnection(StreamingHttp
                         // ignore
                     }
                 })
-                .expectError()
-                // .thenCancel()
+                // FIXME: use thenCancel() after await() instead of cancelling from inside then(...) + expectError()
```

Comment:
can you create an issue and link it here?

Reply:
https://github.com/apple/servicetalk/issues/1492
Also added this link for `FIXME` comments: https://github.com/apple/servicetalk/pull/1491/commits/43dff7cf63df3fa914e492fb4ebe3dcdb017bfc5